src/modules/RootPart.rb

```diff
@@ -291,8 +290,8 @@ def AddMountedPartition(partinfo)
 
     # Check the filesystem of a partition.
     def FSCKPartition(partition)
-      detected_fs = probed.partitions.with(name: partition).filesystems.first.to_s.to_sym
-      if detected_fs == :ext2
+      detected_fs = fstype_for_device(probed, partition)
+      if detected_fs == "ext2"
```

Comment:
I considered this but the existing code is basically expecting a string and I didn't want to change the code unnecessarily. We still have to merge with master at some time.